Userland/Libraries/LibJS/Runtime/AtomicsObject.cpp

```diff
@@ -46,36 +46,42 @@ static ThrowCompletionOr<ArrayBuffer*> validate_integer_typed_array(VM& vm, Type
     return buffer;
 }
 
-// 25.4.2.2 ValidateAtomicAccess ( typedArray, requestIndex ), https://tc39.es/ecma262/#sec-validateatomicaccess
+// 25.4.2.2 ValidateAtomicAccess ( typedArray, requestIndex ), https://tc39.es/proposal-resizablearraybuffer/#sec-validateatomicaccess
```

Comment:
For proposals that override an existing methods, keep both spec links and numbers for grepability

```cpp
// 25.4.2.2 ValidateAtomicAccess ( typedArray, requestIndex ), https://tc39.es/ecma262/#sec-validateatomicaccess
// 6.2.2 ValidateAtomicAccess ( typedArray, requestIndex ), https://tc39.es/proposal-resizablearraybuffer/#sec-validateatomicaccess
```